internal/security/secretstoreclient/common.go

```diff
@@ -0,0 +1,99 @@
+/*******************************************************************************
```

Comment:
The common doRequest() and tests have been refactored into common.go and common_test.go.